src/almanac/introduction.md

```diff
@@ -0,0 +1,3 @@
+# Welcome
+
+Welcome to the Software Gardening Almanac, an open-source handbook of applied guidance and tools for sustainable software development and maintenance.
```

Comment:
```suggestion
Welcome to the Software Gardening Almanac, an open-source handbook of applied guidance and tools for sustainable scientific software development and maintenance.
```

Optional suggestion - do we want to specify here?

Reply:
I went to the grant application and see you pulled this verbatim (which is great!) - so feel free to ignore my suggestion please.

Reply:
Thanks for this! I can see how this could be a helpful clarification. I also feel torn; my personal feeling is that the content should be presented at some level to a common audience which avoids exclusivity to achieve better utility (similar to a farmer's almanac available in a grocery store or public library, able to be picked up by anyone). There's a scientific spirit which will be channeled through the content. Just the same, I don't want the language here to unintentionally convey that some but not all are welcome due to formal titles (if only through self-perception).